docs/docs/guides/integrate/identity-providers/azure-ad.md

```diff
@@ -0,0 +1,112 @@
+3. Choose "Web" in the redirect uri field and add the URL:
+ - {your-domain}/ui/login/login/externalidp/callback
+ - Example redirect url for the domain `https://acme-gzoe4x.zitadel.cloud` would look like this: `https://acme-gzoe4x.zitadel.cloud/ui/login/login/externalidp/callback`
+5. Save the Application (client) ID and the Directory (tenant) ID from the detail page
+
+![Azure App Registration](/img/guides/azure_app_registration.png)
+
+![Azure Client ID and Tenant ID](/img/guides/azure_client_tenant_id.png)
+
+### Add client secret
+
+To be able to authenticate your Azure application you have to generate a new client secret.
+
+1. Click on client credentials on the detail page of the application or use the menu "Certificates & secrets"
+2. Click on "+ New client secret" and enter a description and an expiry date, add the secret afterwards
+3. Copy the value of the secret. You will not be able to see the value again afterwards 
+
+![Azure Client Secret](/img/guides/azure_client_secret.png)
+
+### Token configuration
+
+To allow ZITADEL to get the information from the authenticating user you have to configure what kind of optional claims should be returned in the token.
+
+1. Click on Token configuration in the side menu
+2. Click on "+ Add optional claim"
+3. Add email, family_name, given_name and preferred_username to the id token
+
+![Azure Token configuration](/img/guides/azure_token_configuration.png)
+
+### API persmission
+
+To be able to get all the information that ZITADEL needs, you have to configure the correct permissions.
+
+1. Go to "API permissions" in the side menu
+2. Make sure the permissions include "Microsoft Graph": email, profile and User.Read
+3. The "Other permissions granted" should include "Microsoft Graph: openid"
+
+![Azure API permissions](/img/guides/azure_api_permissions.png)
+
+## ZITADEL Configuration
+
+### Create new Azure AD Provider
+
+Go to the settings of your ZITADEL instance or the organization where you like to add a new **Azure AD** provider.
+Choose the **Microsoft** provider template. This template has everything you need preconfigured. You only have to add the client ID and secret, you have created in the step before.
+
+You can configure the following settings if you like, a useful default will be filled if you don't change anything:
+
+**Scopes**: The scopes define which scopes will be sent to the provider, `openid`, `profile`, and `email` are prefilled. This information will be taken to create/update the user within ZITADEL. Make sure to alswo add `User.Read`
+
+**Email Verified**: Azure AD doesn't send the email verified claim in the token of the user, if you don't enable this setting, the user will be created with the email not verified, which results in a Email Verification Message for the user. If you want to avoid that, make sure to enable "Email verified". In that case the user will be created with a verified email address.
```

Comment:
I'd use more present tense, split sentences where possible and avoid "of the":

```suggestion
**Email Verified**: Azure AD doesn't send the email verified claim in the users token, if you don't enable this setting.
The user is then created with an unverified email, which results in an email verification message.
If you want to avoid that, make sure to enable "Email verified".
In that case, the user is created with a verified email address.
```